.eslintrc.yaml

```diff
@@ -1,7 +1,8 @@
-parser: "babel-eslint"
+#parser: "babel-eslint"
 
 extends:
   - "@saritasa/eslint-config-base"
```

Comment:
It would better to remove this line, cause `@saritasa/eslint-config-react` extends `@saritasa/eslint-config-base`